test/test_tensor_creation_ops.py

```diff
@@ -1382,6 +1414,42 @@ def test_meshgrid(self, device):
         self.assertTrue(grid_b2.equal(expected_grid_b))
         self.assertTrue(grid_c2.equal(expected_grid_c))
 
+    def test_meshgrid_ij_indexing(self, device):
```

Comment:
In the future it'll be interesting to compare against NumPy directly, too

Reply:
we do that in the OpInfo via reference. Is there a different flavor of comparison we should do here?

Reply:
SampleInputs are usually a small set of valid inputs because they're used to generate so many tests. A more specific "forward" test is a good way to exercise a wider and more exhaustive range of inputs

Reply:
That makes sense: we only want this heavy volume of inputs for the reference test.

Do you have an example test I can look at so I can gauge how comprehensive to be? I'll write the test in the meantime.

Reply:
It should exhaustively test the operator's code paths and stress edge case behavior. The exact number of cases is a little difficult to prescribe in advance. For some references here's the test cases for nextafter:

https://github.com/pytorch/pytorch/blob/1c2b5e59ae53ef3042ad1b5cf9aa275391c92971/test/test_binary_ufuncs.py#L1875

and here's for einsum:

https://github.com/pytorch/pytorch/blob/d37636901ed1c65c1f8b68e36e37e59eb503c554/test/test_linalg.py#L4493

Reply:
implemented!